
Reported upstream - just a couple of missing TCP_INFO guards in a change
made in 1.7.4.2

diff -wpruN '--exclude=*.orig' a~/filan.c a/filan.c
--- a~/filan.c	1970-01-01 00:00:00
+++ a/filan.c	1970-01-01 00:00:00
@@ -897,12 +897,14 @@ int tcpan(int fd, FILE *outfile) {
       ++optname;
    }
 
+#ifdef TCP_INFO
    tcpan2(fd, outfile);
+#endif
    return 0;
 }
 #endif /* WITH_TCP */
 
-#if WITH_TCP
+#if WITH_TCP && defined(TCP_INFO)
 
 int tcpan2(int fd, FILE *outfile) {
    struct tcp_info tcpinfo;
